packages/js/github-actions/actions/eslint-annotation/README.md

```diff
@@ -0,0 +1,38 @@
+# ESLint formatter to annotate the linting results in GitHub Actions
+
+This action provides the following functionality for GitHub Actions users:
+
+- Annotate the errors and warnings of eslint results in GitHub Actions.
+
+## Usage
+
+See [action.yml](action.yml)
+
+## Compatibility
+
+- `eslint` >= 7
+
+#### Basic:
+
+```yaml
+name: ESLint
+
+on:
+  pull_request:
+
+jobs:
+  eslint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16
+      - run: npm ci
+      - uses: woocommerce/grow/eslint-annotation@actions-v1
+        with:
+          formatter-dest: "./my-formatter.cjs"
+      - run: eslint --format ./my-formatter.cjs src
```

Comment:
I don't understand well this `formatter-dest` option.

Following the code I see that we copy `eslintFormatter.cjs` file in the path  indicated in `formatter-dest` (`./my-formatter.cjs`) and then use that file for the formatting. 

Why not to call always those files `eslintFormatter.cjs` ???

Reply:
The action intends to call the GitHub annotation command via the formatter and still prints the original eslint results. To achieve this, `eslint` is not bundled in the script of this action but uses the same version of the caller repo. Therefore, the `eslintFormatter.cjs` script needs to be placed under the directory of caller repo to import its `eslint` package dynamically.

Reply:
Ahh, ok, so we are copying the `eslintFormatter.cjs` (always named like that) into a file defined as `formatter-dest ` indicates. 

Is that correct? If so, why is necessary to have an optionfor naming that "script_dest" ??